eng/CredScanSuppression.json

```diff
@@ -24,6 +24,35 @@
     {
       "hash": "5ETIzLYe9aa9Xx5512Uy6gJiCZ/CL2QzS3nDfN51nz8=",
       "_justification": "Secret used by test code"
-    }
+    },
+    {
+      "placeholder": [
+        "`$`{NPM_TOKEN`}",
+        "azure_client_secret&scope=https%3A%2F%2Fattest.azure.net%2F.default",
```

Comment:
These don't exactly look like placeholders

Reply:
It does not include any generated string. Just follow the format of the key. so it is still placeholder to me.